example/multiTenancyExampleOnchainPrivacyEventLogs/setup.js

```diff
@@ -0,0 +1,71 @@
+const Web3 = require("web3");
+const fs = require("fs");
+const path = require("path");
+const EEAClient = require("../../src");
+
+const { besu, orion } = require("../keys");
+const { createHttpProvider } = require("../helpers.js");
+
+const bytecode = fs.readFileSync(
+  path.join(__dirname, "../solidity/EventEmitter/EventEmitter.bin")
+);
+
+const node = new EEAClient(
+  new Web3(createHttpProvider(orion.node1.jwt, besu.node1.url)),
+  2018
+);
+
+async function run() {
+  const enclaveKey = orion.node1.publicKey;
+  const addresses = [orion.node1.publicKey, orion.node2.publicKey];
+
+  // create privacy group
+  const onChainPrivacyGroupCreationResult = await node.privx.createPrivacyGroup(
+    {
+      participants: addresses,
+      enclaveKey: orion.node1.publicKey,
+      privateFrom: orion.node1.publicKey,
+      privateKey: besu.node1.privateKey
+    }
+  );
+  console.log("CREATION RESULT");
+  console.log(onChainPrivacyGroupCreationResult);
+
+  await node.priv.getTransactionReceipt(
+    onChainPrivacyGroupCreationResult.commitmentHash,
+    orion.node1.publicKey
+  );
+
+  const { privacyGroupId } = onChainPrivacyGroupCreationResult;
+  console.log(privacyGroupId);
```

Comment:
I think it would be easier for the user if we log something like: "Created privacy group <id>"

Maybe this is what we were doing on line 42?